plugins/headless/hu.bme.mit.gamma.headless.server/.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
+eclipse.preferences.version=1
+resolveWorkspaceProjects=true
+version=1
```

Comment:
`hu.bme.mit.gamma.headless.server/bin` is not in gitignore, all the `*.class` files are uploaded too.